templates/components/itilobject/timeline/timeline_item_header_badges.html.twig

```diff
@@ -31,14 +31,25 @@
  # ---------------------------------------------------------------------
  #}
 
-<span class="badge text-wrap d-none d-md-block">
+{% set timeline_display_date = session('glpitimeline_date_format') %}
+ {# Display tooltip only when relative dates are displayed#}
+{% set toggle_tooltip %}
+   {% if timeline_display_date == constant('Config::TIMELINE_RELATIVE_DATE') %}
+      tooltip
+   {% endif %}
+{% endset %}
+<span class="badge user-select-auto text-wrap d-none d-md-block">
    {% set date_span %}
       <span title="{{ date_creation|formatted_datetime }}"
```

Comment:
Title is is not more relevant as tooltip.
Maybe the solution is to have this kind of code:
```
<span
    {% if timeline_display_date == constant('Config::TIMELINE_RELATIVE_DATE') %}
        title="{{ date_creation|formatted_datetime }}"
        data-bs-toggle="tooltip" data-bs-placement="bottom"
    {% endif %}
>

Reply:
done